rest/src/main/java/pro/taskana/rest/WorkbasketController.java

```diff
@@ -134,9 +134,9 @@
         }
     }
 
-    @RequestMapping(value = "/{workbasketKey}", method = RequestMethod.PUT)
+    @RequestMapping(value = "/{workbasketId}", method = RequestMethod.PUT)
     public ResponseEntity<WorkbasketResource> updateWorkbasket(
-        @PathVariable(value = "workbasketKey") String workbasketKey,
+        @PathVariable(value = "workbasketId") String workbasketKey,
```

Comment:
Sollte das nicht auch workbasketId heißen? Und wäre es nicht geschickt, wenn wir sicherstellen würden, dass der Workbasket in der Payload tatsächlich diese Id hat?